Appraisals

```diff
@@ -0,0 +1,52 @@
+appraise 'rails-3.1.x' do
```

Comment:
This is cool! I didn't know about appraisal, seems like exactly what this project needs.